sdk/search/search-documents/src/synonymMapHelper.browser.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SynonymMap } from "./serviceModels";
+
+/**
+ * Helper method to create a SynonymMap object. This is a NodeJS only method.
+ * Will throw an error for browser.
+ *
+ * @param _name - Name of the SynonymMap.
+ * @param _filePath - Path of the file that contains the Synonyms (seperated by new lines)
+ * @returns SynonymMap object
+ */
+export async function createSynonymMapFromFile(
+  _name: string,
```

Comment:
@xirzec  I have marked the parameters with `_` as they are not used. Is this acceptable?

Reply:
Good enough for me!